frontend/js/bubble_chart.js

```diff
@@ -564,3 +564,86 @@ d3.csv('data.csv', display);
 
 // setup the buttons.
 setupButtons();
+
+//JS associated with the potatoes
+/*var AN_click = 0;
+var AP_click = 0;
+var SD_click = 0;
+var CO_click = 0;
+var HA_click = 0;*/
+
+var an, ap, sd, co, ha;
+an = document.getElementById('AN_pot');
+ap = document.getElementById('AP_pot');
+sd = document.getElementById('SD_pot');
+co = document.getElementById('CO_pot');
+ha = document.getElementById('HA_pot');
+
+function AN_clicked(){
+  clearAllBubbleStyles();
+  chart.selectNodes = function () {
+// TODO: select only the angry dreams
+    bubbles.each()
+          .attr('stroke', "#BF1238")
+          .attr('stroke-width', 1);
+  };
+  an.style.color = "#000";
+}
+
+function AP_clicked(){
```

Comment:
These functions can be factorized, this code will need to be refactored in another PR